client/src/app/store.tsx

```diff
@@ -0,0 +1,170 @@
+import React from "react";
+import StoreCard from "@/components/layout/StoreCard";
+
+interface Store {
+  storeName: string;
+  storeRating: number;
+  storeLogo: {
+    logoImage: string;
+    imageAlt: string;
+  };
+  storeCategories: string[];
+  storeItems: {
+    itemImage: string;
+    imageAlt: string;
+  }[];
+}
+
+const stores: Store[] = [
```

Comment:
I would put example data into a separate file and import it to keep your files neater. That also makes for easy an edit when incorporating data from BE eventually. Example data files could go inside a folder as a "related file" like I described in my comment at the beginning of this file when talking about directory structure